.changeset/quick-yaks-poke.md

```diff
@@ -0,0 +1,6 @@
+---
+"@twilio-paste/chat-composer": patch
+"@twilio-paste/core": patch
+---
+
+[ChatComposer] removed internal unused imports
```

Comment:
is this referring to the story files? If so it's not necessary to make a changelog for it, not a change to the package

Reply:
No it was a for imports in the chat Composers src but looking at the files after rebase they aren't in the changes so I'll remove